model/src/order.rs

```diff
@@ -460,6 +474,42 @@ impl Default for OrderKind {
     }
 }
 
+/// Location for which the sellAmount should be drawn upon order fulfilment
+#[derive(Eq, PartialEq, Clone, Copy, Debug, Deserialize, Serialize, Hash, enum_utils::FromStr)]
+#[enumeration(case_insensitive)]
+#[serde(rename_all = "snake_case")]
+pub enum BalanceFrom {
+    /// Order owner's ethereum address
+    Owner,
```

Comment:
I think these should match the SC enum, as that is what you will use to sign:
- `erc20`
- `external`
- `internal`